packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.scss

```diff
@@ -39,4 +39,10 @@
 
 :host([uswds]) .usa-breadcrumb__list-item:not(:last-child)::after {
   background: var(--color-gray-medium);
+}
+
+@media screen and (min-width: 768px) {
+  :host(:not([uswds])) ul {
+    padding-left: 0;
```

Comment:
Does this have to have a min-width? Or do we just not want the left padding for all viewports? If so, could you just remove it from the original CSS?

![Screenshot 2023-09-22 at 8 59 09 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/85b3e448-1c7d-45d5-9458-d462ef218a90)

Reply:
Do you mean "max-width" here? On larger views, we don't need the left padding. However, for smaller screens, I've observed that the left padding is necessary.

Reply:
Honestly, I believe we shouldn't include left padding at all. Omitting it would offer greater flexibility to developers.

Reply:
oops yes! I meant `padding-left` (not sure why I wrote width).

Reply:
@jamigibbs it would be beneficial to keep it without preset padding and allow developers to add them according to their requirements. What's your take on this?

Reply:
@harshil1793 I feel like that might be a better approach (leave it to teams to adjust padding on the host) but I'm just not really sure why it was added there to begin with. Maybe just double check the Sketch file for this component?

Reply:
Confirmed, No padding in v1.